### ISSUE TITLE
Issues with Spore

### ISSUE THREAD

**Issue by joeyballentine** [NVIDIAGameWorks/rtx-remix#379]
I've done various tests with Spore and have noticed a few interesting issues.

First thing of note: Spore's graphics are entirely shader-based. That being said, it does sorta work with remix. However, there are many issues with it.

The most stable part I've tested is the creature creator:

![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/34788790/9973e0db-a5a5-41fd-97ad-5a4747afd175)

Once you get into creature stage though, the first main issue you will probably notice is that many objects are missing textures and seem to be entirely colored by their vertex colors. This is most noticeable on the terrain. All creatures also are mostly transparent with only certain features being shown based on the creature's paint job.

![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/34788790/7d64d042-4928-4a9a-aa48-e71f8c2b725b)

In previous playtests (months ago), I was able to get the terrain to sort of show up by setting the corrupted textures in the remix texture selector to "sky" textures. This made the terrain have texture again, but weirdly inverted its depth and made it semi-transparent. 

![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/34788790/f61e19d8-03b9-40f8-93ca-968e13acf60b)

Now, the most interesting issue in all of this is actually the terrain generation getting messed up, You can see in the first screenshot above (with the blue/purple terrain) that the terrain has a giant wall nearby. That is not supposed to happen in normal gameplay.

More importantly, the terrain completely changes each time you save and re-open the level. I also noticed today that this even occurs when you press `alt+enter` to toggle fullscreen. The terrain just drastically changes with compounding corruptions. 

Here is an example of the map in a world before i saved+reloaded:
![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/34788790/92c28ee0-5f8e-426e-9a04-c6235b8c3739)

and after:
![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/34788790/eb4be17f-4e18-43ab-a52d-516a5988b08b)

and after again:
![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/34788790/a4e997d5-40b0-4b59-aa6f-7509a5acb2c2)

My assumption here is that remix uses shaders in some way to track terrain data, and that breaks due to whatever shader interceptions remix is doing. You can even see in the remix dev textures menu there are many corrupted textures (which are what I set to sky textures in the above screenshot with the semi-working terrain). These textures mainly look like vertex explosions, but in 2d. These textures are not consistent btw, and do change each load (consistent with each time the terrain breaks). 

I'm hoping that fixing the underlying cause of this might indirectly end up improving compatibility with other games, as it seems to me like remix is unintentionally doing something that directly interferes with how a game works (which it obviously shouldn't be able to do)

There are a variety of other issues that make Spore just not really viable for use with Remix, but nonetheless I wanted to report this in case it can help other games or potentially make Spore compatible with remix in the future.

## Game setup
1. Copy the remix files to both the root spore directory as well as to `/SporeBin`
2. Use this rtx.conf for convenience where I've already selected many of the UI textures: [rtx.conf.txt](https://github.com/NVIDIAGameWorks/rtx-remix/files/14073491/rtx.conf.txt). If you don't use this, the most important thing to do is enable in-view transform.
3. Run the game in windows xp compatibility mode
4. In game, turn the graphics settings all the way down
5. To get into creature stage quickly, press `ctrl+shift+c` and enter the cheat `levels -unlock`. Then make a new game and select creature stage

## Context
1. This is the steam version of the game
2. I'm using the latest CI remix builds (but these issues have been consistent for a long time)

If you have any questions or want me to try anything, do let me know.

**Comment by NV-LL** [comment]
Hello! Will you please test this with the latest release and let us know if the issue persists? Thank you!

**Comment by joeyballentine** [comment]
@NV-LL Yes. All these issues still exist in the latest release of both the bridge and runtime.
